Templates/CDS-CSharp/SecurityHelper.csx

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Runtime.Remoting.Messaging;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.CommonDataService;
+using Microsoft.CommonDataService.Configuration;
+using Microsoft.CommonDataService.ServiceClient.Security;
+using Microsoft.CommonDataService.ServiceClient.Security.Credentials;
+
+internal class SecurityHelper
+{
+    public static Task<Client> GetClientFromConfiguration(HttpRequestMessage requestMessage, TraceWriter logger)
+    {
+        var settings = new ConnectionSettings
+        {
+            Tenant = EnvironmentValues.Tenant,
+            EnvironmentId = EnvironmentValues.EnvironmentId,
+            Credentials = new UserImpersonationCredentialsSettings()
+            {
+                ApplicationId = EnvironmentValues.ApplicationId,
+                RedirectUri = EnvironmentValues.RedirectUri,
+                ApplicationSecret = EnvironmentValues.ApplicationSecret
+            }
+        };
+
+        return settings.CreateClient(requestMessage, logger as TraceWriterTelemetryBridge ?? new TraceWriterTelemetryBridge());
```

Comment:
See the EnvironmentValues class in run.csx. We eventually want to replace it with the auth info that we will get from the ambient context.[https://github.com/rakpar/azure-webjobs-sdk-templates/blob/rakpar-cds/Templates/CDS-CSharp/run.csx](url)